src/media/js/main.js

```diff
@@ -61,7 +62,7 @@ define('main', [
     });
 
     z.page.on('navigate', function(e, url) {
-        if ([urls.reverse('login'), urls.reverse('fxa_authorize')].indexOf(url)) {
+        if ([urls.reverse('login'), urls.reverse('core/fxa_authorize')].indexOf(url)) {
```

Comment:
I made a `permissions` module and added some tests. It's probably worth porting that to MCM at some point. This has some logic to redirecting to `/curate/403` if the user isn't authorized so that should probably get incorporated first. I just hardcoded URLs so that will need to change.